main.js

```diff
@@ -0,0 +1,127 @@
+const apiKey = '59193db5e99e491a9098d88600213ffa'
+
+const button = document.getElementById('button');
+
+const eventFunction = function (){
+  const city = document.querySelector('#search-query').value;
+  getCoordinates(city);
+  document.querySelector('#search-query').value = "";
+}
+
+button.addEventListener('click', eventFunction)
+document.querySelector('#search-query').addEventListener('keypress', function(event){
+  if (event.key === "Enter"){event.preventDefault(); eventFunction()}
+})
+
+const fetchWeather = function(lat, lon){
+  const url = `https://api.openweathermap.org/data/2.5/weather?lat=${lat}&lon=${lon}&appid=${apiKey}&units=imperial`
+  fetch(url, {
+    method: 'GET',
+    dataType: 'json'
+  })
+  .then(data => data.json())
+  .then(data => {
+    renderCurrentWeather(data.name, data.main.temp, data.weather[0].main)
+    renderIcon(data.weather[0].icon)
+  });
+}
```

Comment:
Be sure to end these types of lines with semi-colons. Looks like you are missing on all your functions.